microcosm_pubsub/consumer.py

```diff
@@ -76,6 +76,8 @@ def nack(self, message, visibility_timeout_seconds=None):
     limit=10,
     # SQS will only return a few messages at time unless long polling is enabled (>0)
     wait_seconds=1,
+    # By default, don't change message visibility on nack
+    visibility_timeout_seconds=None,
```

Comment:
Housekeeping: This config was previously defined in the consumer daemon's defaults instead of in the defaults of the relevant component.